client/my-sites/domains/domain-management/list/domains-table.jsx

```diff
@@ -143,15 +143,14 @@ class DomainsTable extends PureComponent {
 
 			// TODO: how can we optimize the data loading? Can we load the daomin data using `InView` component as in list-all.jsx?
 			return (
-				<>
+				<Fragment key={ `${ domain.name }-${ index }` }>
```

Comment:
We need to set the key on the fragment instead of on the `DomainRow` component to avoid the unique key warning.